src/wallet/onchain.ts

```
@@ -112,4 +117,72 @@
+        // Sign inputs and Finalize
+        child = await this.identity.sign(child);
+        for (let i = 1; i < child.inputsLength; i++) {
+            child.finalizeIdx(i);
+        }
+
+        try {
+            await this.provider.broadcastTransaction(parent.hex, child.hex);
+        } catch (error) {
+            console.error(error);
+        } finally {
+            return [parent.hex, child.hex];
```

Biome (1.9.4)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)